packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -90,7 +95,9 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
             xyData: {},
             columns: [{title: 'Name', sortable: true}],
             allMax: 0,
-            allMin: 0
+            allMin: 0,
+            cursor: 'default',
```

Comment:
Should a constant be defined and reused for this repeating 'default' value?

Reply:
Fixed.

Reply:
How about CURSOR_DEFAULT and so on, or one constant per reused cursor string value?